File: Surface/Surface/SfSize.m
//
//  SfSize.m
//  Surface
//
//  Created by Webchimp on 22/03/16.
//  Copyright © 2016 Webchimp. All rights reserved.
//

#import "SfSize.h"

@implementation SfSize


- (id) init
{
    self = [super init];
    if (self) {
        
        self.width = 0;
        self.height = 0;
    }
    return self;
}

- (void)setSize:(float)width andHeight:(float)height {
    
    self.width = width;
    self.height = height;
}

- (BOOL)isEmpty {
    
    if( !self.width && !self.height ) {
        return true;
    }
    return false;
}

@end

File: Swift/V2/SfSize.swift
//
//  SfSize.swift
//  Surface Swift
//
//  Created by Webchimp on 26/09/16.
//  Copyright © 2016 AppBuilders. All rights reserved.
//

import Foundation

class SfSize: NSObject {
    
    var width:Float! = 0;
    var height:Float! = 0;
    
    func setSize(width:Float!, height:Float!) {
        
        self.width = width;
        self.height = height;
    }
    
    func isEmpty() -> Bool {
        
        if ( self.width == nil && self.height == nil ) {
            return true;
        }
        
        return false;
    }

}
